indico/modules/events/timetable/controllers/legacy.py

```diff
@@ -73,6 +75,29 @@ def _process(self):
         return jsonify_form(form)
 
 
+class RHLegacyTimetableAddSessionBlock(RHManageTimetableBase):
+    def _checkParams(self, params):
+        RHManageTimetableBase._checkParams(self, params)
+        self.session = Session.get_one(request.args['session'])
```

Comment:
`self.session = self.event_new.get_session(request.args['session'])` (you need to check that it's not none manually)

Right now you could pass a session id from another event... and since it's not part of the url but just the query string url normalization won't help here